.github/workflows/publish_release.yml

```diff
@@ -0,0 +1,68 @@
+name: Publish Stable Cymetric
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - '*'
```

Comment:
Still not clear if this is invoked with a push to `main` without any tags?

Reply:
I think `release = released` may be safe too?  Tagging could happen during drafting a release, but this only happens when a release if published as non-draft.  At least that's my interpretation from: https://docs.github.com/en/webhooks/webhook-events-and-payloads?actionType=released#release

Reply:
I think you're right, I had been misinterpreting the docs.  Updated trigger in `publish_release.yml`